test/libsolidity/SolidityCompiler.cpp

```diff
@@ -29,7 +29,19 @@ using namespace std;
 namespace solidity::frontend::test
 {
 
-BOOST_FIXTURE_TEST_SUITE(SolidityCompiler, AnalysisFramework)
+class SolidityCompilerFixture: protected AnalysisFramework
+{
+	void setupCompiler(CompilerStack& _compiler) override
+	{
+		AnalysisFramework::setupCompiler(_compiler);
+
+		// FIXME: This test was probably supposed to respect CommonOptions::get().optimize but
+		// due to a bug it was always running with optimizer disabled and it does not pass with it.
+		_compiler.setOptimiserSettings(false);
```

Comment:
The check that fails is:
```c++
BOOST_CHECK(creationBytecode.size() >= 90);
```

The internal functions are probably getting optimized out and the code should be adjusted.